diff --git a/minicluster/src/main/java/org/apache/accumulo/minicluster/impl/MiniAccumuloClusterImpl.java b/minicluster/src/main/java/org/apache/accumulo/minicluster/impl/MiniAccumuloClusterImpl.java
index e512936..b78e958 100644
--- a/minicluster/src/main/java/org/apache/accumulo/minicluster/impl/MiniAccumuloClusterImpl.java
+++ b/minicluster/src/main/java/org/apache/accumulo/minicluster/impl/MiniAccumuloClusterImpl.java
@@ -713,7 +713,8 @@ public class MiniAccumuloClusterImpl implements AccumuloCluster {
 
   @Override
   public ClientConfiguration getClientConfig() {
-    return new ClientConfiguration(Arrays.asList(new MapConfiguration(config.getSiteConfig()))).withInstance(this.getInstanceName()).withZkHosts(
+    MapConfiguration array = new MapConfiguration((Map<String, Object>)(Map<String, ?>)config.getSiteConfig());
+    return new ClientConfiguration(Arrays.asList(array)).withInstance(this.getInstanceName()).withZkHosts(
         this.getZooKeepers());
   }
 
diff --git a/pom.xml b/pom.xml
index 3d0d903..8c3d880 100644
--- a/pom.xml
+++ b/pom.xml
@@ -179,7 +179,7 @@
       <dependency>
         <groupId>commons-configuration</groupId>
         <artifactId>commons-configuration</artifactId>
-        <version>1.6</version>
+        <version>1.9</version>
       </dependency>
       <dependency>
         <groupId>commons-httpclient</groupId>
